### ISSUE TITLE
[cli] Unable to install some helm v3 apps from local directory

### ISSUE THREAD

**Issue by rancher-max** [rancher/rancher#27916]
**What kind of request is this (question/bug/enhancement/feature request):**
feature request

**Description:**
Need to be able to pass helm version when installing apps from local directory. By default it uses `rancher-helm` (or helm-v2) which will fail for some apps that require helm v3

**Comment by deniseschannon** [comment]
This has been solved with the addition of Helm 3 type of catalogs. Can we verify it's working with the CLI?

**Comment by brendarearden** [comment]
We have provided the helm version option when using the cli to add a catalog https://github.com/rancher/cli/blob/d2f200ca7c76c3cb115d19eb48f1155fc55a544a/cmd/catalog.go#L91 , but since can install an app from a local directory through the cli, there is no helm version on the app when it attempts to install, which will then use the default helm version to install with. https://github.com/rancher/cli/blob/d2f200ca7c76c3cb115d19eb48f1155fc55a544a/cmd/app.go#L157 The ability to change the helm version for just apps still needs to be added.

**Comment by deniseschannon** [comment]
It sounds like the workaround would be to add the apps from your local directory as a helm 3 catalog and deploy from the catalog?

**Comment by brendarearden** [comment]
Yes, if the app is part of a catalog, then the helm version will propagate to the app when you deploy it from the catalog. We do not support local directory catalogs (as far as I can tell), so the charts would have to be pushed up some form of version control or hosted catalog.
